yellowme/youst-kotlin
# Move animation xml files to core

## What does this PR do?

- Remove anim files from miniapps and playground module.
- Add anim files to core. 
- Add R app import to DeepLinkWidgetProvider.
- With this changes the app bundle build should succeed.

## How should this be manually tested?

Now it should allow the build of the app bundles.

Go to `Build` -> `Build Bundle(s) / APK(s)` ->  `Build Bundle(s)`, try to build the bundles, this action should succeed.

## What are the relevant tickets?

Closes Issue #30 

## Screenshots
N/A
